drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dropdown/ScenarioSimulationScesimFilesDropdownProviderKogitoImpl.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.dropdown;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import javax.enterprise.context.Dependent;
+
+import org.kie.workbench.common.widgets.client.assets.dropdown.KieAssetsDropdownItem;
+import org.kie.workbench.common.widgets.client.assets.dropdown.KieAssetsDropdownItemsProvider;
+
+@Dependent
+public class ScenarioSimulationScesimFilesDropdownProviderKogitoImpl implements KieAssetsDropdownItemsProvider {
+
+    @Override
+    public void getItems(Consumer<List<KieAssetsDropdownItem>> assetListConsumer) {
+        List<KieAssetsDropdownItem> toAccept = new ArrayList<>();
+        assetListConsumer.accept(toAccept);
```

Comment:
Can't be just `assetListConsumer.accept(Collections.emptyList())`?

Reply:
@jomarko No, because `Collections.emptyList()` is immutable. At the beginning that list is empty, but it could be have items added later by the user, then I need an empty mutable list. I just removed the local variable.